pages/dashboard.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../css/normalize.css">
+    <link rel="stylesheet" href="../css/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Plus+Jakarta+Sans:wght@400;500;600;700;800&display=swap" rel="stylesheet">
+    <title>Dashboard</title>
+</head>
+<body>
+    <div id="dashboard-root">
+    <header>
+        <nav>
+            <!-- <img src="" alt="" class="logo-btn"> -->
+            <p class="nav__logo">Logo</p>
+            <div class="nav__right">
+            <img src="../../images/setting.png" alt="" class="nav__right__settings-btn">
+            <div class="nav__right__global-menu">
+                <img src="../images/global-menu.svg" alt="" class="nav__right__global-menu-btn">
+            </div>
+            <div class="nav__right__global-menu--open">
+                <button class="global__menu__close-btn"><img src="../images/close-btn.svg" alt=""></button>
+                <p class="global__menu__service"><a href="to-do.html" onclick="route()">To-do List</a></p>
+                <p class="global__menu__service"><a href="pomodoro.html" onclick="route()">Pomodoro</a></p>
+                <p class="global__menu__service"><a href="time-tracker.html" onclick="route()">Time Tracker</a></p>
+                <p class="global__menu__service"><a href="sticky-notes.html" onclick="route()">Sticky Notes</a></p>
+            </div>
+            <div class="nav__right__user-profile">
+                <p class="nav__right__user-profile__user-name">Jasleen</p>
+                <img src="../images/arrow-down.png" alt="" class="nav__right__user-profile__dropdown-btn">
```

Comment:
Why not svg for this one too?

Reply:
As this one was available in the assets of the design. So, used it as it is.
Shall I add svg icon for it too?

Reply:
Yes, I think other icons are in svg format. So better to be consistent.